backend/src/main/java/ai/verta/modeldb/project/FutureProjectDAO.java

```diff
@@ -48,7 +50,7 @@ public FutureProjectDAO(Executor executor, FutureJdbi jdbi, UAC uac) {
     tagsHandler = new TagsHandler(executor, jdbi, entityName);
   }
 
-  public InternalFuture<Void> deleteAttributes(DeleteExperimentRunAttributes request) {
+  public InternalFuture<Void> deleteAttributes(DeleteProjectAttributes request) {
```

Comment:
Unrelated change

Reply:
This Pr is for the Project entity and this method is foe delete project attributes not the experiment run so this is the valid change as per PR heading.

Reply:
No, it is not. It is a bad copy and paste from before. I'm _really_ trying to remove the bad practice of adding a bunch of unrelated changes to the same PR as it makes software much less robust.

Reply:
Yes. Fixed on the base PR